jevan-de/observer/main.py

```diff
@@ -0,0 +1,19 @@
+from logger import Logger
+from sys import stderr, stdout
+from log_aggregator import LogAggregator
+
+
+def main() -> None:
+    aggr = LogAggregator()
```

Comment:
Any ideas for what other patterns would be useful in making the aggregator instance easily available throughout the whole program?

Reply:
I guess you could use a Singleton as well.

Reply:
Probably you would want to use the same instance everywhere, unless you want to have a unique subscriber set for each instance. Singleton is very common yeah, many frameworks are using dependency injection and/or service locator for this kind of thing.